Sample/sdk-ios-swift/Profile.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Profile.swift
+//  sdk-ios-swift
+//
+//  Created by Simon Hamadene on 15/10/2018.
+//  Copyright © 2018 Luke Ashley-Fenn. All rights reserved.
+//
+
+import Foundation
+
+struct ProfileDictionary: Decodable {
+    let attributes: [Profile]
```

Comment:
Naming is confusing, please keep them in sync

Reply:
And better be documented that is just our only sample implementation.
The structure is up to the client

Reply:
@casperkclee : Yes, it says Sample app. I think it is clear enough that this implementation is "our implementation." and not a template on how to create an app using the SDK.

Reply:
@simon-hamadene I have to agree with @casperkclee request for documentation on that one, as I faced the same issue on previous SDK I've worked on.

Reply:
I have added a comment inside the code.